src/lib/mina.ts

```diff
@@ -581,6 +588,7 @@ function LocalBlockchain({
     },
     getActions(
       publicKey: PublicKey,
+      _actionHashes: ActionHashes,
```

Comment:
I'm afraid that ignoring the `_actionHashes` input here leads to inconsistent behavior between the `Mina.Network` and `Mina.LocalBlockchain` interfaces. Maybe you could move the logic in `SmartContract.getActions` which filters the actions here instead?

Reply:
Moved the logic over! :D